recipe/emailverification/api/emailverify.go

```diff
@@ -51,7 +51,7 @@ func EmailVerify(apiImplementation evmodels.APIInterface, options evmodels.APIOp
 			return supertokens.BadInputError{Msg: "The email verification token must be a string"}
 		}
 
-		response, err := (*apiImplementation.VerifyEmailPOST)(token.(string), options, &map[string]interface{}{})
+		response, err := (*apiImplementation.VerifyEmailPOST)(token.(string), options, supertokens.MakeDefaultUserContextFromAPI(options.Req))
```

Comment:
this file has another API call as well which you missed. Please make sure to properly check all the APIs and add the default context.